MetaMask/metamask-extension
# feat: Support various redesign signatures in ConfirmPage Storybook

## **Description**

This PR adds support for various redesign signatures for the ConfirmPage storybook. 
[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/MetaMask/metamask-extension/pull/25054?quickstart=1)

## **Related issues**

Fixes: https://github.com/MetaMask/metamask-extension/issues/25052

## **Manual testing steps**

1. Run storybook: `yarn storybook`
2. Go to ConfirmPage
3. Observe storybook pages

## **Screenshots/Recordings**



### **Before**

![CleanShot 2024-06-05 at 16 23 29@2x](https://github.com/MetaMask/metamask-extension/assets/20778143/aa2463b1-860b-40d2-b6d8-78366eb7a091)

### **After**

![CleanShot 2024-06-05 at 18 33 20@2x](https://github.com/MetaMask/metamask-extension/assets/20778143/010f2724-d754-492a-ada0-2eab5f28e037)

https://github.com/MetaMask/metamask-extension/assets/20778143/12a1742d-5eb7-43e1-849f-7b388cc4967d

higher quality of above: 
https://github.com/MetaMask/metamask-extension/assets/20778143/342a6182-c443-4bd9-907a-3441b4b9eb81



## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
